### PR TITLE
Formik. Validation

### PR DESCRIPTION
**Pull request by melk0sha**
**Task 7. Forms**

Installed formik.
Performed implementation of “Add movie” form with validation.
Performed implementation of “Edit movie” form with validation.
Used hooks from formik.

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -1,19 +1,7 @@
...
-      (genresAcc, genre) =>
-        moviesAcc.findIndex((accValue) => accValue.value === genre) === -1
-          ? [...genresAcc, { id: getHashCodeFromString(genre), value: genre }]
-          : genresAcc,
-      []
-    );
-
-    return [...moviesAcc, ...uniqueGenresWithIds];
-  }, []);
+export const getGenreId = (genreValue) => genres.find((genre) => genre.value === genreValue).id;
```
Comment (cate-blint) [inline src/utils/index.js]: what happens if find() finds nothing?

File: src/components/shared/alert/alert.styled.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+export const AlertText = styled.span`
+  color: ${({ theme }) => theme.colors.white};
+  font-size: 1rem;
+  margin-bottom: 20px;
+`;
+
+// export const StyledButton = styled(Button)`
+//   margin-top: 30px;
+// `;
```
Comment (cate-blint) [inline src/components/shared/alert/alert.styled.js]: is it necessary here?

Comment (melk0sha) [reply src/components/shared/alert/alert.styled.js]: deleted)

File: src/utils/index.js (hunk 1 above)
Comment (melk0sha) [reply src/utils/index.js]: added alternative for id

**Pull request merged**